src/components/interactive-map.js

```diff
@@ -24,11 +25,11 @@ const propTypes = Object.assign({}, StaticMap.propTypes, {
   minPitch: PropTypes.number,
 
   /**
-   * `onChangeViewport` callback is fired when the user interacted with the
+   * `onViewportChange` callback is fired when the user interacted with the
```

Comment:
I think `onChange` is quite generic in this context. I can't think of anything more specific than `onViewportChange` but will be happy to change it if we come up with something!